gear-programs/vft-treasury/src/services/mod.rs

```diff
@@ -0,0 +1,422 @@
+use collections::HashMap;
+pub(super) use error::Error;
+use msg_tracker::{MessageInfo, MessageStatus, MessageTracker, TxDetails};
+use sails_rs::{gstd::debug, gstd::ExecContext, prelude::*};
+mod bridge_builtin_operations;
+pub mod error;
+mod msg_tracker;
+mod token_operations;
+mod utils;
+mod vft;
+
+pub struct VftTreasury<ExecContext> {
+    exec_context: ExecContext,
+}
+
+static mut DATA: Option<VftTreasuryData> = None;
+static mut CONFIG: Option<Config> = None;
+static mut MSG_TRACKER: Option<MessageTracker> = None;
+
+#[derive(Debug, Default)]
+struct VftTreasuryData {
+    admin: ActorId,
+    receiver_contract_address: H160,
+    gear_bridge_builtin: ActorId,
+    ethereum_event_client: ActorId,
+    bridging_payment_service: ActorId,
+    eth_to_vara_token_id: HashMap<H160, ActorId>,
+    vara_to_eth_token_id: HashMap<ActorId, H160>,
```

Comment:
I think it'd be better to store just `Vec<H160, ActorId>` and perform mapping using linear search. It's not significantly slower and will allow not to maintain 2 mappings